pkg/controller/pxc/users.go

```diff
@@ -92,6 +100,41 @@ func (r *ReconcilePerconaXtraDBCluster) reconcileUsers(cr *api.PerconaXtraDBClus
 	return nil
 }
 
+func (r *ReconcilePerconaXtraDBCluster) manageMonitorUser(cr *api.PerconaXtraDBCluster, internalSysSecretObj *corev1.Secret) error {
+	annotationName := "grant-for-1.6.0-monitor-user"
+	if internalSysSecretObj.Annotations[annotationName] == "done" {
+		return nil
+	}
+
+	rootPass, ok := internalSysSecretObj.Data["root"]
```

Comment:
why root? we agreed to use "operator" user everywhere if it exists 😞

Reply:
done https://github.com/percona/percona-xtradb-cluster-operator/pull/548/commits/5c4e6d5cae55f1578a25f81abaaf8f3c20093fef